src/LogicApp/LogicApp/Properties/Resource.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AssemblyParameterNeedsToBeSpecified" xml:space="preserve">
```

Comment:
Doesn't look like this first string is ever used, please remove.

Reply:
You are correct. It used to be used when I was doing ParameterSets incorrectly. Removed